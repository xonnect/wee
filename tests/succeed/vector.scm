(define x '#(2 4 6 8 9))
(displayln (vector-ref x 2))
(displayln (bitwise-xor (vector-ref x 2) (vector-ref x 4)))
(define y (vector 1 2 3 4))
(displayln (vector-ref y 2))
(vector-set! y 2 7)
(displayln (vector-ref y 2))
(displayln (+ (car x) (vector-ref x 1) (vector-ref y 3)))
