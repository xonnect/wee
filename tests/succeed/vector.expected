6
15
3
7
10
